src/formio/components/SoftRequiredErrors.stories.js

```diff
@@ -0,0 +1,111 @@
+import {expect, userEvent, waitFor, within} from '@storybook/test';
+
+import {withUtrechtDocument} from 'story-utils/decorators';
+import {sleep} from 'utils';
+
+import {UPLOAD_URL, mockFileUploadDelete, mockFileUploadPost} from './FileField.mocks';
+import {MultipleFormioComponents} from './story-util';
+
+export default {
+  title: 'Form.io components / Custom / SoftRequiredErrors',
+  decorators: [withUtrechtDocument],
+  render: MultipleFormioComponents,
+  args: {
+    components: [
+      {
+        type: 'file',
+        key: 'file',
+        storage: 'url',
+        url: UPLOAD_URL,
+        label: 'Soft required file',
+        multiple: false,
+        openForms: {softRequired: true},
+      },
+      {
+        type: 'textfield',
+        key: 'textfield',
+        label: 'Soft required text',
+        openForms: {softRequired: true},
```

Comment:
Textfield cannot be `softRequired`, right? I thought this was only for file upload fields :)

Reply:
officially they can't (yet), but the implementation was meant to be generic so that it also works for text fields :wink:

Reply:
Ah nevermind.. I'm seeing your comment in the Textfield.spec.js 👍